package.json

```diff
@@ -1,8 +1,8 @@
 {
 	"description": "Liferay Frontend Guidelines",
 	"devDependencies": {
-		"eslint": "^6.8.0",
-		"eslint-config-liferay": "^20.0.1",
+		"eslint": "7.9.0",
```

Comment:
Fun fact: this is the first repo (I think) that we upgrade to ESLint 7.9.0; I haven't looked at the changelog, but I'm secretly relieved that it didn't break anything.

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia1.giphy.com%2Fmedia%2F3KC2jD2QcBOSc%2Fgiphy.gif&token=party" data-gitmeme-token="party"><img src="https://media1.giphy.com/media/3KC2jD2QcBOSc/giphy.gif" title="Created by gitme.me with /party"/></a>